fem/restriction.cpp

```diff
@@ -168,8 +168,8 @@ void ElementRestriction::TAddMultTranspose(const Vector& x, Vector& y) const
          for (int j = offset; j < next_offset; ++j)
          {
             const int idx_j = (d_indices[j] >= 0) ? d_indices[j] : -1 - d_indices[j];
-            dof_value += ((d_indices[j] >= 0) ? d_x(idx_j % nd, c, idx_j / nd) :
-                          -d_x(idx_j % nd, c, idx_j / nd));
+            dof_value += (d_indices[j] >= 0) ? d_x(idx_j % nd, c, idx_j / nd) :
+            -d_x(idx_j % nd, c, idx_j / nd);
```

Comment:
The old version seems more clearly indented to me

Reply:
I agree, this seems to have been a weird `astyle` behavior which got triggered by removing the redundant outer parentheses. Reverted in [f75ece6](https://github.com/mfem/mfem/pull/3526/commits/f75ece6427d65532bd6b954f0c8bb6a697f3fbd2).